ci-operator/config/openshift/cluster-storage-operator/openshift-cluster-storage-operator-master.yaml

```diff
@@ -0,0 +1,37 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: origin-v4.0
+    namespace: openshift
+    tag: base
+canonical_go_repository: github.com/openshift/cluster-storage-operator
+images:
+- from: base
+  to: cluster-storage-operator
+resources:
+  '*':
+    limits:
+      cpu: '2'
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: origin-v4.0
+  namespace: openshift
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    namespace: openshift
+    name: release
+    tag: golang-1.10
+tests:
+- as: unit
+  commands: make test
+  container:
+    from: src
+- as: verify
+  commands: make verify
+  container:
+    from: src
```

Comment:
you need the aws e2e job here

Reply:
okay, added. does this `Secrets` test focus I've copied run after my image has been built & manifests applied to the cluster? Or before, as some kind of sanity check?

Reply:
@wongma7: this test type (`openshift_installer`) runs the installer to deploy a cluster with the release images + the images that were built from the PR, then runs the ginkgo test(s).